### DISCUSSION TITLE
Issue with Blog authors

### DISCUSSION THREAD

**Discussion by RossKen** [squidfunk/mkdocs-material#5735] [Q&A] [answered]
Hello!

First of all, thank you so much for all the work put into this project! I am a maintainer of an open-source python package, [Splink](https://github.com/moj-analytical-services/splink), and mkdocs-material has been invaluable! 🎉

Given the (pre)release of Piri Piri, I have been playing around with the Blog plugin. However, I have been having issues with adding authors to the blogs. Having copied across the `hello-world.md` and `.authors.yml` from the [blog docs](https://squidfunk.github.io/mkdocs-material/setup/setting-up-a-blog/#adding-authors), I get the following error when running `mkdocs serve`

```
ERROR    -  Blog post 'blog/posts/hello-world.md' author 'squidfunk' unknown, not listed in .authors.yml
```
I have tried to create a [minimal example](https://github.com/RossKen/mkdocs-material-blog/tree/master), and that is working without error. So, I am struggling to work out what exactly is going wrong. 

I imagine it is a simple error on my end, but I can't figure out why I am getting the above error. Any advice/guidance would be much appreciated!

To reproduce error:

**mkdocs-material version:** mkdocs-material-9.2.0b1
**Related PR:** https://github.com/moj-analytical-services/splink/pull/1451/files

```
git clone git@github.com:moj-analytical-services/splink.git
cd splink
git checkout 1215-blog
source scripts/make_docs_locally.sh
```

where [`scripts/make_docs_locally.sh`](https://github.com/moj-analytical-services/splink/blob/master/scripts/make_docs_locally.sh) is a simple bash script which builds a virtual environment and runs `mkdocs serve`

**Comment by squidfunk** [comment]
My pleasure, glad you give the beta and blog plugin a spin!

Also, thanks for taking the time to create a minimal reproduction. If the minimal reproduction works without errors, then you could build your blog from there? Check that the file location is correct, and that the format of `.authors.yml` is correct.

**Comment by RossKen** [comment, marked as answer]
Thanks @squidfunk - turns out it was a simple error when copying across from the minimal example. Apologies for taking up your time - and thanks again for a superb tool!

**Reply by squidfunk** [reply]
No worries, glad you could solve it quickly!
